paperqa/readers.py

```diff
@@ -13,6 +13,15 @@
 from paperqa.utils import ImpossibleParsingError
 from paperqa.version import __version__ as pqa_version
 
+try:
```

Comment:
I think we can prob just make this required

Reply:
Oh this isn't supposed to be here, let me revert

Reply:
Reverted, sorry